doc/Introduction_to_TF24_IVIM-MRI_CodeCollection_github_and_IVIM_Analysis_using_Python.ipynb

```diff
@@ -0,0 +1,1107 @@
SUKioqFBkZaXY5AAAgRLT2HoI+DwAAtFVN7SGszXnT4uJi3XTTTQ0ev/HGG1VSUtKctwQAAEAQoM8DAABoXLNCtLi4OOXn5zd4PD8/X2efffYpFwUAAIDAos8DAABoXLPmRHvwwQc1YcIErV+/XqmpqW5zZdjtdq1Zs0YvvviiXwoFAACA/9DnAQAANK5ZIdp1112nnj17Ki8vT48//ni9VZvWr1+v5ORkvxQKAAAA/6HPAwAAaFyzQjRJGjZsmIYNG+aPWgAAAGAi+jwAAICGNTtE8+S9995TVVWVkpOTdcYZZ/jiLQEAABAE6PMAAAB+1KwQ7dFHH9XRo0f10EMPSZIMw1B6err++c9/SpK6d+8uu92u8847z/eVAgAAwG/o8wAAABrXrNU5X3nlFZ1//vmu7ddee00bNmzQ+++/r8OHD+uiiy7SAw880Owili5dqri4OEVERCgpKUmbN29ucOwPP/ygBx98UH379lVERIQSExO1Zs2aZn8mAAAA/stffR4AAEBr0awQbc+ePbrwwgtd26tXr9avfvUrXXrpperSpYtmz56twsLCZhXwyiuvKDMzU/PmzdPWrVuVmJiotLQ0HTx40OP42bNn68knn9TixYu1fft2TZ06Vddee60++eSTZn0uAAAA/ssffR4AAEBr0qwQ7cSJEwoPD3dtFxYWuk0+26NHDx0+fLhZBeTk5GjKlCmaPHmyBgwYoGXLlqlDhw5avny5x/EvvPCC7r//fo0ePVrnnHOO7rjjDo0ePVqPP/54sz4XAAAA/+WPPg8AAKA1aVaI1rdvX23YsEGSVFpaquLiYl122WWu4//+97915plnNvn9ampqtGXLFqWmpv63IKtVqampDf7SWV1drYiICLd97du318aNGxscX1lZ6fZqDXILipVnL/F4LM9eotyC4gBXBAAAQpmv+zwAAIDWplkh2l133aVp06bp17/+tdLT05WcnKwBAwa4jq9bt06DBg1q8vsdPnxYDodD0dHRbvujo6NVVlbm8Zy0tDTl5OSopKRETqdTBQUFWrlypQ4cOOBxfHZ2tqKiolyv2NjYJtcXzGxWi3I8BGl59hLlFBTLZrWYVBkAAAhFvu7zAAAAWptmhWhTpkxRXl6evv32W1122WV6/fXX3Y5//fXXuvXWW31a4MkWLVqk+Ph4nXvuuQoLC9O0adM0efJkWa2ev0pWVpYqKipcr3379vm1vkDJSIlX5qgEtyCtNkDLHJWgjJR4kysEAAChJBj6PAAAgGBmMQzDaOrgdevW6fLLL5fNZvPJh9fU1KhDhw567bXXNG7cONf+SZMm6ciRI3rzzTcbPPf48eP65ptv1KNHD82cOVP/+Mc/9Pnnn3v9zMrKSkVFRamiokKRkZG++Bqmqg3OwmxW1TicBGgAAPhJa+shTubrPs8Mrf1vBAAA/KOpPUSz7kS77bbb1K1bN02YMEGvvvrqKc8vFhYWpiFDhshut7v2OZ1O2e12JScnN3puRESEevbsqRMnTuj111/XNddcc0q1hKqMlHhXgBZmsxKgAQCAFvF1nwcAANDaNCtE2717t9avX68BAwboT3/6k6KjozVq1CgtXrxYpaWlLSogMzNTTz/9tJ577jl98cUXuuOOO1RVVaXJkydLkiZOnKisrCzX+E2bNmnlypXavXu33n//fV111VVyOp269957W/T5oS7PXuIK0GoczgYXGwAAAGiMP/o8AACA1qRZIZokXXjhhZo9e7Y2b96sL7/8Uv/zP/+jd955R/369dPAgQM1d+5cffzxx01+v/Hjx+tPf/qT5s6dq4EDB6qoqEhr1qxxLTZQWlrqtmjA8ePHNXv2bA0YMEDXXnutevbsqY0bN6pz587N/Sohr+4caMUL0uvNkQYAANAcvu7zAAAAWpNmzYnWmKqqKq1Zs0ZvvvmmVq9erczMTN1///2+eGufai1zZTS0iACLC4Sm3J9WVPX0N8uzl8jhNDRjVIIJlQEAarWWHqIl6PMAAEBr1tQeol1z3rSwsFDffPONfvGLX7j2Pf/885o3b56qqqo0btw4Pf3002rXrp2+/fbbllcPrxxOw2NQVrvtcPokG0WA2KwW5RQUS1KDoSgAAP5EnwcAANC4Zj3O+eCDD7qtgPnZZ5/p17/+tVJTUzVz5ky9/fbbys7Ols1mU7du3XxeLP5rRiN3mmWkxHPXUojJSImv9zgudxUCAAKJPg8AAKBxzboTraioSA899JBr++WXX1ZSUpKefvppSVJsbKzmzZun+fPn+7RIoC2oDcpyCoq1ZN0u1TicBGgAgIChzwMAAGhcs+5E+89//uOa8F+S3nvvPaWnp7u2L774Yu3bt8931QFtTEZKvGul1TCblQANABAw9HmAf+Q2svBXnr1EuT9N6QEACH7NCtGio6O1Z88eSVJNTY22bt2qSy65xHX8u+++02mnnebbCoE2JM9e4grQahxOVloFAAQMfR7gH7Vz357c19VO3WGzWkyqDADQXM16nHP06NGaOXOmHn30Ua1atUodOnTQiBEjXMc//fRT9e3b1+dFAm3ByXOg1W5L4o40AIDf0ecB/lF3yo7abea+BYDQ1Kw70R566CG1a9dOl19+uZ5++mk9/fTTCgsLcx1fvny5rrzySp8XCbR2nhopT4sNAADgL/7q85YuXaq4uDhFREQoKSlJmzdvbnT8woUL1a9fP7Vv316xsbGaMWOGjh8/3uzPBYJJ3b4uYdY7BGgAEKKadSda165dtWHDBlVUVKhjx46y2Wxux//+97+rY8eOPi0QaAscTsNjI1W77XAaZpQFAGhD/NHnvfLKK8rMzNSyZcuUlJSkhQsXKi0tTTt37lT37t3rjX/xxRc1c+ZMLV++XMOGDVNxcbFuueUWWSwW5eTknNL3A8yWkRLvWjyKuW8BIDRZDMNoU//feWVlpaKiolRRUaHIyEizywEAACGCHqL5kpKSdPHFF2vJkiWSJKfTqdjYWN19992aOXNmvf
```

Comment:
I was able to run it in a colab with some changes up until this point. https://colab.research.google.com/drive/12FETqIZy2yOUceNwWDHca1b2OGZc1ATH?usp=sharing